cmake/gsLibrary.cmake

```diff
@@ -213,6 +213,15 @@ set_target_properties(${PROJECT_NAME} PROPERTIES
 
 endif(GISMO_BUILD_LIB)
 
+list (FIND GISMO_OPTIONAL "gsOpennurbs" _index)
+if (${_index} GREATER -1)
+  # find_package(opennurbs REQUIRED)
+  include(${CMAKE_SOURCE_DIR}/external/gsOpenNurbs.cmake)
+
+  # TODO Check found if not cmake error
+  target_link_libraries(${PROJECT_NAME} ${OPENNURBS_LIBRARY_PATH})
```

Comment:
The value of OPENNURBS_LIBRARY_PATH seems to be problematic on Windows
https://cdash-ci.irisa.fr/viewBuildError.php?buildid=152858

For some reason the file libopennurbs.a is searched..